ee-feature-pack/channel/pom.xml

```diff
@@ -29,12 +35,12 @@
 
         <!-- We need the feature pack built first as that generates the manifest -->
         <dependency>
-            <groupId>${project.groupId}</groupId>
+            <groupId>org.wildfly</groupId>
```

Comment:
This should be ${ee.maven.groupId}.

The original ${project.groupId} was wrong but harmless since it will have the same value as ${ee.maven.groupId}, even downstream. But hardcoded org.wildfly will break downstream.

Reply:
The problem is that the `${ee.maven.groupId}` is replaced by `${project.groupId}` which becomes `org.wildfly.channel` because of the channel project groupId.

Reply:
OK; we should fix that but that's out of scope for this PR. :)